codespell_lib/data/dictionary.txt

```diff
@@ -1650,6 +1664,22 @@ ambuguity->ambiguity
 ambulence->ambulance
 ambulences->ambulances
 amdgput->amdgpu
```

Comment:
Not sure of the origin of this one, since it isn't from this PR, but it looks to me like it should be... 

amdgput->AMD GPU

... instead.   
?